drivers/sdhc/Kconfig.brcm

```diff
@@ -21,14 +21,22 @@ config SDHC_BUFFER_ALIGNMENT
 
 choice BRCM_BCM2712_XFER_MODE
 	bool "Select BCM2712 SDHC (SD/MMC) data transfer mode"
-	default BRCM_BCM2712_XFER_SDMA
+	default BRCM_BCM2712_XFER_ADMA
 
 config BRCM_BCM2712_XFER_PIO
 	bool "BCM2712 SDHC PIO data transfer mode"
 
 config BRCM_BCM2712_XFER_SDMA
 	bool "BCM2712 SDHC SDMA data transfer mode"
 
+config BRCM_BCM2712_XFER_ADMA
```

Comment:
Why isn't the name `BRCM_BCM2712_XFER_ADMA2`?
I don't see that you are trying to enable modes other than ADMA2.

Also, in the case where you have ADMA2 32-bit descriptor mode, the alignment (`SDHC_BUFFER_ALIGNMENT`) can be 32 bits.